### PR TITLE
Add crowdfunding donors page to website

### PR DESCRIPTION
**Pull request by amakarudze**
We promised our crowdfunding supporters mention of their names in newsletter and on our website. These changes add a page to the website with the list of our awesome crowdfunding supporters who chose to be visible donors during the campaign.

### CODE REVIEW

File: core/templatetags/element_by_counter.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from django import template
+register = template.Library()
+
+@register.simple_tag
+def element_by_counter(data, counter, divider):
+     index = int(counter / divider)
```
Comment (asendecka) [inline core/templatetags/element_by_counter.py]: Let's make sure we handle `divider = 0` well. If it's `0` we should return empty string, too, instead ending up with division by zero error.

File: templates/core/crowdfunding_donors.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                <img src="{% static 'img/global/donate/django_crowdfunding_heart.png' %}"></a>
+            </div>
+        </div>
+
+        <br/>
+
+        <div class="row">
+            <h2>Our awesome supporters ✨</h2>
+            {% for donor in donor_list %}
+                 {% if forloop.first %}<div class="row">{% endif %}
```
Comment (asendecka) [inline templates/core/crowdfunding_donors.html]: This line is not indented nicely - there is one extra space I think. It's super small thing, but it's nice to have nicely formatted code :D. Let's remove one space before `{%`

File: templates/includes/_main_sponsors.html [hunk 3]
```diff
@@ -104,4 +104,14 @@
             <img class="img-responsive center-block" src="{% static 'img/global/supporters/wsp.png' %}" alt="WSP Digital" />
         </a>
     </div>
+    <div class="col-md-3">
```
Comment (asendecka) [inline templates/includes/_main_sponsors.html]: Is there any reason we add extra sponsors here? Are they sponsors who donated through Crowdfunding?

File: core/templatetags/element_by_counter.py [hunk 4]
```diff
@@ -0,0 +1,9 @@
+from django import template
+register = template.Library()
+
+@register.simple_tag
+def element_by_counter(data, counter, divider):
```
Comment (asendecka) [inline core/templatetags/element_by_counter.py]: Could you add tests to this template tag? Could be in separate PR as we don't want to block this to be merged.

File: templates/core/crowdfunding_donors.html [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+            <div class="col-md-6">
+                <p>
+                This year (2018) we ran a crowdfunding campaign on Indiegogo's generosity website from
+                the 29th January to the 29th of March with the aim of raising US $10,000. We are truly
+                grateful to all our awesome supporters who helped reach our target goal. We managed to
+                raise a total of $13,314 - 133% of our goal!
+                </p>
+            </div>
+            <div class="col-md-6">
+                <img src="{% static 'img/global/donate/django_crowdfunding_heart.png' %}"></a>
```
Comment (asendecka) [inline templates/core/crowdfunding_donors.html]: Is this image added to the site?

File: templates/core/crowdfunding_donors.html [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+            {% for donor in donor_list %}
+                 {% if forloop.first %}<div class="row">{% endif %}
+                <div class="col-md-4">
+                    <h4 class="donor-text">{{ donor.name }}</h4>
+                </div>
+                {% if forloop.counter|divisibleby:3 %}</div><div class="row">{% endif %}
+                {% if forloop.last %}</div>{% endif %}
+                {% if forloop.counter|divisibleby:42 %}
+                    <br />
+                    {% element_by_counter quotes forloop.counter 42 as quote %}
```
Comment (asendecka) [inline templates/core/crowdfunding_donors.html]: `forloop.counter` starts from `1`, not `0` - that means that first quote will never be displayed. Instead we should use `forloop.counter0` (but only in this line!)

File: templates/includes/_main_sponsors.html (hunk 3 above)
Comment (amakarudze) [reply templates/includes/_main_sponsors.html]: Yes, they are new sponsors from crowdfunding campaign as well as new sponsors who came on board after the campaign.

File: core/templatetags/element_by_counter.py (hunk 4 above)
Comment (amakarudze) [reply core/templatetags/element_by_counter.py]: Will work on this. Thanks

File: core/templatetags/element_by_counter.py (hunk 1 above)
Comment (amakarudze) [reply core/templatetags/element_by_counter.py]: Done, thanks.

File: templates/core/crowdfunding_donors.html (hunk 2 above)
Comment (amakarudze) [reply templates/core/crowdfunding_donors.html]: Fixed indentation.
